src/selectors/publish.js

```diff
@@ -244,20 +243,6 @@ export const getSanitizedProfileData: Selector<Promise<Uint8Array>> =
     Promise.resolve().then(() => compress(serializeProfile(profile)))
   );
 
-/**
- * The blob is needed for both the download size, and the ObjectURL.
- */
-export const getCompressedProfileBlob: Selector<Promise<Blob>> = createSelector(
-  getSanitizedProfileData,
-  async (profileData) =>
-    new Blob([await profileData], { type: 'application/octet-binary' })
-);
-
-export const getDownloadSize: Selector<Promise<string>> = createSelector(
-  getCompressedProfileBlob,
-  (blobPromise) => blobPromise.then((blob) => prettyBytes(blob.size))
-);
```

Comment:
I moved these behaviors to the components. It's used only there currently and I don't see the need in keeping them in the state. This also simplifies the handling because there's just one promise to unwrap then.